packages/webview_flutter/example/lib/main.dart

```diff
@@ -34,6 +34,30 @@ class WebViewExample extends StatelessWidget {
     );
   }
 
+  Widget listButton() {
+    return FutureBuilder<WebViewController>(
+      future: _controller.future,
+      builder:
+          (BuildContext context, AsyncSnapshot<WebViewController> controller) {
+        if (controller.hasData) {
```

Comment:
We shouldn't hide the entire menu before the controller is available (e.g "Make a toast" is independent of a controller availability).
Maybe just disable the "Evaluate Javascript" disabled before we have a controller?